pkg/sensors/tracing/generickprobe.go

```diff
@@ -290,6 +297,10 @@ func createMultiKprobeSensor(sensorPath, policyName string, multiIDs []idtable.E
 		}
 		oneKprobeHasStackTrace = oneKprobeHasStackTrace || gk.hasStackTrace
 		oneKprobeHasRatelimit = oneKprobeHasRatelimit || gk.hasRatelimit
+		// it should be overkill (we could just get the value of the first)
+		// because the caller should have already uniformized hasFDInstall
+		// between all the kprobes
+		resizeFDInstall = resizeFDInstall || gk.hasFDInstall
```

Comment:
could we just pass the detected oneKprobeHasFDInstall in createMultiKprobeSensor directly?
maybe we don't need to store it for kprobeEntry, but just pass it to createKprobeSensorFromEntry and createMultiKprobeSensor as single bool ?

Reply:
yep yep we can, it's another way of passing that stuff. I thought it would clobber the params of the functions but if that's simpler I'll try.

Reply:
this is much more reasonable and readable indeed https://github.com/cilium/tetragon/compare/1b4c684aa4b475b380a433dfa7cde7217b179b0c..f0189d37149a202690251a3d84e1b95fc4f87657